locksmith/src/utils/keyPricer.ts

```diff
@@ -56,6 +57,16 @@ export default class KeyPricer {
     if (lock.keyPrice === '0') {
       return 0
     }
+
+    const { creditCardPrice } = await lockSettingOperations.getSettings({
+      lockAddress,
+      network,
+    })
+
+    if (creditCardPrice) {
+      return parseInt((creditCardPrice * 100).toFixed(0)) // returns the USD cents price
```

Comment:
CAn you confirm the same format is used by `priceConversion.convertToUSD` ?

Reply:
Yes, is the same format!